libretroshare/src/pqi/pqissludp.cc

```diff
@@ -50,38 +51,21 @@ static const uint32_t PQI_SSLUDP_DEF_CONN_PERIOD = 300;  /* 5  minutes? */
 /********** PQI SSL UDP STUFF **************************************/
 
 pqissludp::pqissludp(PQInterface *parent, p3LinkMgr *lm) :
-	pqissl(NULL, parent, lm), tou_bio(NULL),// listen_checktime(0),
-	mConnectPeriod(PQI_SSLUDP_DEF_CONN_PERIOD), mConnectFlags(0),
-	mConnectBandwidth(0)
-{
-	RS_STACK_MUTEX(mSslMtx);
-
-	sockaddr_storage_clear(remote_addr);
```

Comment:
I believe that this is used to zero the memory in these structs, that have no initialization otherwise.